deepr/prepros/combinators.py

```diff
@@ -58,27 +58,35 @@ def __init__(
         num_parallel_calls: int = None,
     ):
         super().__init__()
-        self.preprocessors = (
-            _fuse(*to_flat_tuple(preprocessors), num_parallel_calls=num_parallel_calls)
-            if fuse
-            else to_flat_tuple(preprocessors)
-        )
+        self.preprocessors = to_flat_tuple(preprocessors)
         self.fuse = fuse
         self.num_parallel_calls = num_parallel_calls
 
+        # Iterable of preprocessors used by the apply method
+        self._preprocessors = (
```

Comment:
keep both the original list of preprocessors and the fused version for nested unrolling